src/validate.js

```diff
@@ -2000,6 +2006,7 @@ function F_VALIDATE(disabledHL) {
 					&& otherSegment.$type === roadType
 					&& otherSegment.$isToll === isToll
 					&& otherSegment.$hasRestrictions === hasRestrictions
+					&& otherSegment.$flags === flags
```

Comment:
Have a look at `deepCompare()` in `helpers.js`

Reply:
Yes I'm using that now. Thanks for the hint